specification/web/resource-manager/Microsoft.Web/stable/2020-09-01/ResourceProvider.json

```diff
@@ -0,0 +1,1398 @@
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/validate": {
+      "post": {
+        "summary": "Validate if a resource can be created.",
+        "description": "Description for Validate if a resource can be created.",
+        "operationId": "Validate",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "validateRequest",
+            "in": "body",
+            "description": "Request with the resources to validate.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ValidateRequest"
+            }
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ValidateResponse"
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "../2020-06-01/CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/validateMoveResources": {
+      "post": {
+        "summary": "Validate whether a resource can be moved.",
+        "description": "Description for Validate whether a resource can be moved.",
+        "operationId": "ValidateMove",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "moveResourceEnvelope",
+            "in": "body",
+            "description": "Object that represents the resource to move.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CsmMoveResourceEnvelope"
+            }
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "No Content"
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "../2020-06-01/CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AppserviceGithubToken": {
+      "description": "Github access token for Appservice CLI github integration.",
+      "type": "object",
+      "properties": {
+        "accessToken": {
+          "description": "Github access token for Appservice CLI github integration",
+          "type": "string"
```

Comment:
Since access token should be treated as a secret, please add `"x-ms-secret": true`.

Reply:
I'm getting validation error saying secret cannot be sent as a response when I add this, but getting this access token is the purpose of this endpoint
![image](https://user-images.githubusercontent.com/14339314/99842741-40731400-2b25-11eb-9af1-e0afeaacfe8a.png)

Reply:
POST can return a secret. Looks like we need to update validation. It's okay not to have x-ms-secret annotation for now.